docs/sources/enterprise/saml/about-saml.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'About SAML authentication in Grafana'
+menuTitle: 'About SAML authentication'
+description: 'SAML authentication'
+keywords: ['grafana', 'saml', 'documentation', 'saml-auth']
+aliases: ['/docs/grafana/latest/auth/saml/']
+weight: 20
+---
+
+# SAML authentication
+
+SAML authentication integration allows your Grafana users to log in by using an external SAML 2.0 Identity Provider (IdP). To enable this, Grafana becomes a Service Provider (SP) in the authentication flow, interacting with the IdP to exchange user information.
+
+The SAML single sign-on (SSO) standard is varied and flexible. Our implementation contains a subset of features needed to provide a smooth authentication experience into Grafana.
+
+> Only available in Grafana Enterprise v6.3+. If you encounter any problems with our implementation, please don't hesitate to contact us.
+
+## Supported SAML
+
+Grafana supports the following SAML 2.0 bindings:
+
+- From the Service Provider (SP) to the Identity Provider (IdP):
+
+  - `HTTP-POST` binding
+  - `HTTP-Redirect` binding
+
+- From the Identity Provider (IdP) to the Service Provider (SP):
+  - `HTTP-POST` binding
+
+In terms of security:
+
+- Grafana supports signed and encrypted assertions.
+- Grafana does not support signed or encrypted requests.
+
+In terms of initiation, Grafana supports:
+
+- SP-initiated requests
+- IdP-initiated requests
+
+By default, SP-initiated requests are enabled. For instructions on how to enable IdP-initiated logins, see https://grafana.com/docs/grafana/latest/enterprise/saml/#idp-initiated-single-sign-on-sso.
+
+## Set up SAML with Okta
+
+This guide will follow you through the steps of configuring SAML authentication in Grafana with [Okta](https://okta.com/). You need to be an admin in your Okta organization to access Admin Console and create SAML integration. You also need permissions to edit Grafana config file and restart Grafana server.
```

Comment:
In this intro, think about what a user who lands on this page would first want to know. Something like "Grafana supports user authentication through Okta, which is useful when you want your users to access Grafana using single signon" - or something like that. Just a bit of text to help the reader become oriented in what they are seeing if they land on this page via a google search.

Also, remember to add a Before you begin section. I can see in the intro references to prerequisites that you can add to the Before you begin section.